README.md

```diff
@@ -62,23 +62,54 @@ An example usage is by getting the list of PMID's from ```get_list_pmid``` and t
 
 ### Command line
 
-Can be run on Command line with the command
+Can be run on Command line with the command, look below for required arguments
 
 ```bash
-pubmed-dl
+main.py --help
 ```
+This prints out:
 
-Program requires two dates to be inputted for start and end
+```bash
+Usage: main.py [OPTIONS] START END REQUEST_FILE
+
+  Main requires 3 arguments.
+
+  The first one is start date (eg: YYYY/MM/DD).
+
+  The second one is end date (eg: 2021/02/05).
+
+  The third one is the filename where the data needs to be written to.
+
+Arguments:
+  START         [required]
+  END           [required]
+  REQUEST_FILE  [required]
+
+Options:
+  --install-completion [bash|zsh|fish|powershell|pwsh]
+                                  Install completion for the specified shell.
+  --show-completion [bash|zsh|fish|powershell|pwsh]
+                                  Show completion for the specified shell, to
+                                  copy it or customize the installation.
+
+  --help                          Show this message and exit.
+```
+
+Program requires three arguments to be inputted by the user. 
+The first one is the start date in the ```YYYY/MM/DD``` format (e.g.: ```2021/02/04```). 
+The second one is the end date in the ```YYYY/MM/DD``` format (e.g.: ```2021/02/05```). 
+The third one is the filename where the data needs to be written to (e.g.: ```test.json```).

```

Comment:
Not really important but I wouldn't copy-paste this here as it's subject to change. I would just say something like "call `pubmed-dl --help` for help.